flink-ai-flow/ai_flow/api/ops.py

```diff
@@ -71,15 +75,20 @@ def write_dataset(input: Channel,
                   name: Text = None
                   ) -> None:
     """
-    Write dataset to dataset operator. It can write dataset to external system.
-
-    :param input: Channel from the specific operator which generates data.
-    :param dataset_info: Information about the dataset which will be read. Its type can be DataSetMeta
-                         of py:class:`ai_flow.meta.dataset_meta.DatasetMeta` or Text or int. The dataset_info
-                         means name in the metadata service when its type is Text and it means id when its type is int.
-                         The ai flow will get the dataset from metadata service by name or id.
-    :param write_dataset_processor: The python user defined function in write dataset operator. User can write their own logic here.
-    :param name: Name of the read_dataset operator.
+    Writes the dataset through the user-defined processor based on the metadata of dataset. The processor writes dataset
+    to external system and generates the :class:`~ai_flow.meta.ai_graph.ai_node.WriteDatasetNode` with
+    :class:`~ai_flow.meta.dataset_meta.DatasetMeta`.
+
+    :param input: The input :class:`~ai_flow.graph.channel.Channel` from the operator that generates the data used to
+                            write dataset.
+    :param dataset_info: Information about the dataset writen. The type of dataset information could be
+                                      :class:`~ai_flow.meta.dataset_meta.DatasetMeta` or Text or int. The `dataset_info`
+                                      means the name of dataset when the type is Text and it means the id of dataset
+                                      when the type is int, which gets the metadata of dataset by the id.or name of
+                                      dataset at default.
```

Comment:
See comments for `read_dataset` method. Maybe the description can be improved following the same format of the comments for `read_dataset`